### ISSUE TITLE
Return value must be an instance of void, none returned

### ISSUE THREAD

**Issue by ulrichdahl** [kreait/firebase-php#129]
#### 1. Which version of `kreait/firebase-php` are you using? 

3.4.1

#### 2. Which version of PHP are you using?

PHP 7.0.16 (cli) (built: Mar  6 2017 19:45:42) ( NTS )
Copyright (c) 1997-2017 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2017 Zend Technologies
    with Zend OPcache v7.0.16, Copyright (c) 1999-2017, by Zend Technologies

#### 3. What's the issue?

Type error: Return value of Kreait\Firebase\Auth\User::setIdToken() must be an instance of Kreait\Firebase\Auth\void, none returned

#### 4. Code that lead to the issue

```php
$this->client = (new Factory)
            ->withServiceAccountAndApiKey($serviceAccount, $apiKey)
            ->withDatabaseUri(config('firebase.databaseURL'))
            ->create();

$this->client = $this->client->asUser(
            $this->client->getAuth()->getUser('some-user-id', ['premiumUser' => true])
);
```

**Comment by ulrichdahl** [comment]
As far as I can gather the `: void` return is a feature of PHP 7.1 but the package states only PHP: ^7.0 requirement.

**Comment by jeromegamez** [comment]
Uh! That's bad! I will fix this and create a new Release tonight. Thank your for taking the time to open this issue!

**Comment by jeromegamez** [comment]
Fixed with 2ec03f7d95a12ea3493e810bff3f1a5fc9fab98a! Thanks again!
